### PR TITLE
More tweaks

### PR DESCRIPTION
**Pull request by artcz**

### CODE REVIEW

File: templates/404.html [hunk 1]
```diff
@@ -1,31 +1,33 @@
...
-            <li>Our <a href="http://blog.europython.eu/">blog</a></li>
-            <li>The <a href="/events/sessions/">conference session list</a></li>
-            <li>Our <a href="/sponsor/">sponsorship opportunities</a></li>
-            <li><a href="/faq/">Frequently asked questions</a> (F.A.Q.)</li>
-            <li>... or get back to the <a href="/">EuroPython Home Page</a></li>
-        </ul>
+                    <p>Here are a few useful links:</p>
+                    <ul>
+                        <li>Our <a href="http://blog.europython.eu/">blog</a></li>
+                        <li>The <a href="/events/sessions/">conference session list</a></li>
```
Comment (umgelurgel) [inline templates/404.html]: I know this wasn't introduced as part of this PR - but the urls (on this line and the following ones) seem to be hardcoded in this template, looks like a good time to replace them with the `{% url ... %}`

**Review by umgelurgel — CHANGES_REQUESTED** [review]
Good work - I see it's still WIP, there's one thing that caught my eye though.

File: templates/404.html (hunk 1 above)
Comment (artcz) [reply templates/404.html]: True. although some of them are {% page_url %} as well. :)

**Comment by artcz** [comment]
Merged it because I fixed the thing pointed in the review. :)

**Comment by artcz** [comment]
Thanks!

**Comment by umgelurgel** [comment]
A bit late, but :shipit: :)

**Pull request merged**